config_cloud.go

```diff
@@ -331,6 +331,5 @@ const (
 
 // DockerConfig stores auth info for Docker.
 type DockerConfig struct {
-	APIVersion    string `bson:"api_version" json:"api_version" yaml:"api_version"`
-	DefaultDistro string `bson:"default_distro" json:"default_distro" yaml:"default_distro"`
+	APIVersion string `bson:"api_version" json:"api_version" yaml:"api_version"`
```

Comment:
Can we flatten this struct into a single string now, something like `DockerVersion`? Or do we anticipate further fields here